src/tests/ftest/aggregation/aggregation_io_small.py

```diff
@@ -0,0 +1,124 @@
+
+        :avocado: tags=all,full_regression,hw,large,aggregate,daosio
+        :avocado: tags=aggregateiosmall
+        """
+
+        # Create pool and container
+        self.update_ior_cmd_with_pool()
+
+        # Since the transfer size is 1K, the objects will be inserted
+        # into SCM
+        scm_index = 0
+        ssd_index = 1
+        block_size = human_to_bytes(self.params.get("block_size",
+                                                    "/run/ior/*"))
+        np = self.params.get("np", "/run/ior/client_processes/*")
+        total_ior = block_size * np
+
+        pool_info = self.pool.pool.pool_query()
+        initial_scm_free_space =\
+            pool_info.pi_space.ps_space.s_free[scm_index]
+        initial_ssd_free_space =\
+            pool_info.pi_space.ps_space.s_free[ssd_index]
+
+        self.log.info("Initial SCM Free Space = {}".format(
+            initial_scm_free_space))
+        self.log.info("Initial SSD Free Space = {}".format(
+            initial_ssd_free_space))
+
+        # Disable the aggregation
+        self.log.info("Disabling the aggregation")
+        self.pool.set_property("reclaim", "disabled")
+
+        # Run ior
+        self.run_ior_with_pool()
+        pool_info = self.pool.pool.pool_query()
+        scm_free_space_after_ior =\
+            pool_info.pi_space.ps_space.s_free[scm_index]
+        ssd_free_space_after_ior =\
+            pool_info.pi_space.ps_space.s_free[ssd_index]
+
+        self.log.info("SCM Free Space after ior = {}".format(
+            scm_free_space_after_ior))
+        self.log.info("SSD Free Space after ior = {}".format(
+            ssd_free_space_after_ior))
+
+        self.log.info("Comparing if scm space after ior - {} is less" \
+            " than initial free space - {}".format(scm_free_space_after_ior,
+                                                   initial_scm_free_space))
+        self.assertTrue(scm_free_space_after_ior <=
+                        (initial_scm_free_space - total_ior))
+
+        self.log.info("Checking that nothing has been moved to SSD")
+        self.assertTrue(ssd_free_space_after_ior == initial_ssd_free_space)
+
+        # Enable the aggregation
+        self.log.info("Enabling the aggregation")
+        self.pool.set_property("reclaim", "time")
+        # wait 90 seconds for files to get old enough for aggregation +
+        # 90 seconds for aggregation to start and finish
+        self.log.info("Waiting for 120 seconds for aggregation to start" \
+            " and finish")
+        time.sleep(180)
+        pool_info = self.pool.pool.pool_query()
+        scm_free_space_after_aggregate =\
+             pool_info.pi_space.ps_space.s_free[scm_index]
+        ssd_free_space_after_aggregate =\
+             pool_info.pi_space.ps_space.s_free[ssd_index]
+
+        self.log.info("Checking the data is moved to SSD after aggregation")
+        self.log.info("{} == {}".format((initial_ssd_free_space - total_ior),
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments